portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,49 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function scrollMe() {
+  var me = document.getElementById('me');
+  me.scrollIntoView(true);
+}
+
+function scrollCat() {
+  var cat = document.getElementById('cat');
+  cat.scrollIntoView(true);
+}
+
+function scrollFavs() {
+  var favs = document.getElementById('favs');
+  favs.scrollIntoView(true);
+}
+
+function scrollGallery() {
+  var gallery = document.getElementById('gallery');
+  gallery.scrollIntoView(true);
+}
+
+function scroll(id) {
+  console.log("scroll to: " + id);
```

Comment:
It’s fine for now but keep in mind we should remove all console logs for production applications :)